### PR TITLE
Рефакторинг

### PR DESCRIPTION
**Pull request by eXqxqxq**
[Review on Reviewable](https://reviewable.io/reviews/basepractice/2015.2/75)

### CODE REVIEW

File: Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package ru.mirea.oop.practice.coursej.s131226.entities;
+
+public class Item {
```
Comment (Pastor) [inline Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java]: Почему файл с маленькой буквой????

Comment (eXqxqxq) [inline Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java]: image
idk wtf

Comment (eXqxqxq) [inline Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java]: image
=(

**Comment by Pastor** [comment]
Пересоздайте файл Item.java

---

Reviewed 43 of 48 files at r1, 5 of 5 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/basepractice/2015.2/75#-:-K6PWzwhHQ3KhQSjioas:-744820059)*

**Comment by Pastor** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java, line 3 \[r1\]](https://reviewable.io:443/reviews/basepractice/2015.2/75#-K6P5zqmX8nnCiO6ym3t-r1-3:-K6PXA5Ihr7cxV6HqSxm:-175328956) ([raw file](https://github.com/basepractice/2015.2/blob/acc48da5a3fab9dd646142383b0afb5b638d4eca/Examples/s131226/src/main/java/ru/mirea/oop/practice/coursej/s131226/entities/item.java#L3)):*
Удалите, закоммитьте и снова создайте

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/basepractice/2015.2/75)*

**Comment by Pastor** [comment]
Оформляйтесь

---

Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/basepractice/2015.2/75#-:-K6Q6IdLW_EFMUmu2ZwI:-426260396)*

**Pull request merged**